### ISSUE TITLE
Can't have more than 3 alts

### ISSUE THREAD

**Issue by NilsTheBest** [WorldLanguages/ScratchNotifier#7]
After entering 3 alts, we can't enter a new one.
http://u.cubeupload.com/NilsTheBest77/AddAltBug.gif

**Comment by WorldLanguages** [comment]
This was in purpose - however I don't see why this needs to stay like this.

**Comment by NilsTheBest** [comment]
Why was it intentional?

**Comment by WorldLanguages** [comment]
I didn't think anyone would need more than 3 alts. But why not make them unlimited?

**Comment by NilsTheBest** [comment]
Perhaps the alts of people who have over 3/4 accounts all serve a purpose!

**Comment by hf3i05** [comment]
> **@NilsTheBest** wrote:
> After entering 3 alts, we can't enter a new one.
> ![addaltbug](https://user-images.githubusercontent.com/41824646/52525372-f3ec7a80-2ca8-11e9-9c6d-b52cb6c2f0a0.gif)
You could just open [ScratchNotifier](https://scratchnotifier.cf) in another browser.

**Comment by locness3** [comment]
And say goodbye to your free ram

**Comment by hf3i05** [comment]
> @locness3 wrote:
> And say goodbye to your free ram

?

**Comment by locness3** [comment]
Running two browsers at the same time for adding alts...

**Comment by hf3i05** [comment]
>@locness3 wrote
>Running two browsers at the same time for adding alts...

You don't need a lot of RAM for that.

**Comment by locness3** [comment]
that's a joke

**Comment by hf3i05** [comment]
> @locness3 wrote
> that's a joke

k

**Comment by wgyt735yt** [comment]
> I didn't think anyone would need more than 3 alts. But why not make them unlimited?

I have to deal with 5 alts, and have to manage accounts for separate projects. I would like this feature added.
![Screenshot 2020-09-03 at 6 47 26 PM](https://user-images.githubusercontent.com/68466727/92180834-f75ccf00-ee15-11ea-8c3d-6767ed53e783.png)

**Comment by williamhorning** [comment]
> That's 8 accounts

Yes... I know. I made all of these accounts. I would have to run chrome, edge, and IE to get all the notifications.

**Comment by moeenio** [comment]
Or use firefox's Container tabs

**Comment by williamhorning** [comment]
> Or use firefox's Container tabs

I don't have firefox :P Might get it soon.

**Comment by williamhorning** [comment]
> 
> 
> > Or use firefox's Container tabs
> 
> I don't have firefox :P Might get it soon.

Firefox saved my life:
![image](https://user-images.githubusercontent.com/68466727/94368344-94c3bf80-00b1-11eb-85c4-f839d29f3912.png)
